PocketKit/Sources/PocketKit/Refresh/FeatureFlagsRefreshCoordinator.swift

```diff
@@ -0,0 +1,78 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Sync
+import SharedPocketKit
+import Combine
+
+/// Refresh coordinator to handle the refreshing of feature flags
+class FeatureFlagsRefreshCoordinator: RefreshCoordinator {
+
+    let taskID: String = "com.mozilla.pocket.refresh.featureFlags"
+
+    let refreshInterval: TimeInterval? = 60 * 60 * 48 // once every 48 hours.
```

Comment:
why setting this as an optional here and then force unwrap it below? Can it just be a constant?